xarray/backends/api.py

```diff
@@ -324,6 +324,8 @@ def open_dataset(
     backend_kwargs=None,
     use_cftime=None,
     decode_timedelta=None,
+    storage_options=None,
```

Comment:
Shouldn't `storage_options` and `fs` be `backend_kwargs`?

Reply:
Happy to move it - this is POC, so I just needed to put it somewhere.
However, it's not exactly a part of the backend kwargs, but it just so happens that it's only used by the zarr backend right now.